src/components/ScholarshipDetailCard.jsx

```diff
@@ -105,7 +96,16 @@ export default function ScholarshipDetailCard({ scholarship }) {
             <Typography component="p">{label}</Typography>
           </Grid>
           <Grid item className={classes.cardDetailText} xs={12} sm>
-            <Typography component="p">{text}</Typography>
+            {link ? (
+              <Link
+                href="#"
+                underline="always"
+                onClick={(e) => e.preventDefault()}>
+                {text}
+              </Link>
+            ) : (
+              <Typography component="p">{text}</Typography>
```

Comment:
Default variant is `body1` which corresponds to `p`.
```suggestion
              <Typography>{text}</Typography>
```